### ISSUE TITLE
Filter-Button: update selected state

### ISSUE THREAD

**Issue by rbascue** [eBay/skin#1737]
We're updating the selected state of the quick filter pill to have a solid border instead of a darker fill.

DOD: component in core matches [Figma](https://www.figma.com/file/zEBdEhbonrBOGzZ0fXzWvM/?node-id=55010%3A91396).

[DT ticket to update Playbook](https://ebay.monday.com/boards/1932639446/pulses/2567507250) for tracking purposes.

![image](https://user-images.githubusercontent.com/35155833/165397936-8f0cbc9f-3342-436f-8035-57ae9ec03e8a.png)
![image](https://user-images.githubusercontent.com/35155833/165398232-44f2bc2e-2c05-4e94-8c41-db11fbc04df1.png)
![image](https://user-images.githubusercontent.com/35155833/165398356-e959469b-2435-4d4d-b4bc-2385494df37b.png)

**Comment by agliga** [comment]
Doesn't have to be next version.
Non breaking
Need to get updated designs for color contrast of the container. 
We can always put this when design is ready, doesn't seem like a big change.

**Comment by rbascue** [comment]
@agliga this is still showing as blocked by design. Is this true? Pushing this update is causing the current selected state to appear much different than this.

**Comment by rbascue** [comment]
If there is a concern with the unselected enabled state, can we add that as a separate ticket and address it there? This ticket is specifically about the selected state's visual treatment.

**Comment by agliga** [comment]
No need to be blocked based on disabled state since we aren't applying color contrast to disabled filter menu items.

**Comment by agliga** [comment]
Randy investigating into adding border. 
We should try to implement this next version to fix the inverse color.
